CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+# Contributing to Atom
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+**OrbitDeterminator** is a community-led project; that means that anyone is welcome to contribute to OrbitDeterminator and join the community! If you would like to make improvements to the package, add new features that are useful to you and others, or have found a bug that you know how to fix, please submit a pull request!
+
+The following is a set of guidelines for contributing to OrbitDeterminator project, which are hosted in the [AerospaceResearch.net organization](https://github.com/aerospaceresearch/orbitdeterminator) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Overview](#overview)
+  * [Getting Started](#getting-started)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [CoffeeScript Styleguide](#coffeescript-styleguide)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [AerospaceResearch.net Code of Conduct](Code_of_Conduct.md). By participating, you are expected to uphold this code.
+
+## I don't want to read this whole thing I just have a question!!!
+
+> **Note:** Please don't file an issue to ask a question. You'll get faster results by using the resources below.
+
+We have an official [mailing list](https://lists.shackspace.de/mailman/listinfo/constellation) where you can post your question for the community.
+
+If chat is more your speed, you can join the Atom and Electron Slack team:
```

Comment:
We don't have this I think.

Reply:
Should I remove Constellation mailing list part??